blueprints/data-solutions/mongodb-cluster/main.tf

```diff
@@ -0,0 +1,235 @@
+/**
```

Comment:
A couple things we might want to do

- split `main.tf` into separate files (and use the tfdoc annotation + configure tfdoc to spit out the file list)
- use the project module with `project_create = false` to enable the required APIs